src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py

```diff
@@ -308,7 +308,7 @@ def test_pf_flow_test_with_non_english_input_output(self, capsys):
         assert output_path.exists()
         with open(output_path, "r") as f:
             outputs = json.load(f)
-            assert outputs["answer"] in stdout
+            assert outputs["answer"] in json.loads(stdout)["answer"]
```

Comment:
why this fix transient issue?

Reply:
The test case failed because of \n in the output answer.  But in stdout, it is \\n, so using json.loads to escaped to \n to solve the transient issue